packages/cardano-services/src/util/DbSyncProvider/DbSyncProvider.ts

```diff
@@ -25,13 +25,20 @@ export const DbSyncProvider = <
     public async healthCheck(): Promise<HealthCheckResponse> {
       try {
         const { ok, localNode } = await this.cardanoNode.healthCheck();
-        const lastBlockNo = (await this.db.query<BlockNoModel>(findLastBlockNo)).rows[0].block_no;
+        const tip = (await this.db.query<LedgerTipModel>(findLedgerTip)).rows[0];
         const isHealthy =
-          ok && lastBlockNo >= (localNode?.ledgerTip?.blockNo ?? DB_MAX_SAFE_INTEGER) - DB_BLOCKS_BEHIND_TOLERANCE;
+          ok && tip.block_no >= (localNode?.ledgerTip?.blockNo ?? DB_MAX_SAFE_INTEGER) - DB_BLOCKS_BEHIND_TOLERANCE;
+
```

Comment:
Let's add a `debug` log message here to reveal the two values used to produce the `isHealthy` value.

Reply:
Now I remember: I thought something similar, but I didn't because `DbSyncProvider` has no `logger` (we already spoken about this and the outcome was that we should refactor/standardize `Provider`s dependencies types in order to add the `logger` here).

Once said that... are we sure that's actually required as the core of this PR is to forward exactly the same data in the output JSON?

Reply:
Please create a ticket to capture the refactor + addition of debug logging

Reply:
ADP-2492 created